### ISSUE TITLE
Unable to install on JupyterLab 1.0.5

### ISSUE THREAD

**Issue by jkbr-19** [karosc/jupyterlab_dracula#1]
Trying to install the theme gives this ouput:
```
$ jupyter labextension install jupyterlab_dracula
An error occured. 
ValueError: This extension does not yet support the current version of JupyterLab. 
Conflicting Dependencies: 
JupyterLab              Extension      Package 
>=1.0.3 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/application 
>=1.0.3 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/apputils 
See the log file for details:  /tmp/jupyterlab-debug-1d74_fys.log 
```
here is the logfile:
```
Node v12.8.0

> /home/jkbr/miniconda3/bin/npm pack jupyterlab_dracula
npm notice 
npm notice 📦  jupyterlab_dracula@0.20.1
npm notice === Tarball Contents === 
npm notice 1.6kB   package.json                                 
npm notice 201B    README.md                                    
npm notice 171B    lib/index.d.ts                               
npm notice 692B    lib/index.js                                 
npm notice 165.7kB static/674f50d287a8c48dc19ba404d20fe713.eot  
npm notice 77.2kB  static/af7ae505a9eed503f8b8e6982036873e.woff2
npm notice 165.5kB static/b06871f281fee6b241d60582ae9369b9.ttf  
npm notice 526.4kB static/embed.css                             
npm notice 98.0kB  static/fee66e712a8a08eef5805a46892932ad.woff 
npm notice 69.3kB  static/index.css                             
npm notice === Tarball Details === 
npm notice name:          jupyterlab_dracula                      
npm notice version:       0.20.1                                  
npm notice filename:      jupyterlab_dracula-0.20.1.tgz           
npm notice package size:  557.0 kB                                
npm notice unpacked size: 1.1 MB                                  
npm notice shasum:        427b727c854c9c2b3615637d7d8a3857465183a0
npm notice integrity:     sha512-oaiwkATXF3y/C[...]m8UAOdSow64Hw==
npm notice total files:   10                                      
npm notice 
jupyterlab_dracula-0.20.1.tgz

Fetching URL: https://registry.npmjs.org/jupyterlab_dracula
Fetching URL: https://registry.npmjs.org/jupyterlab_dracula
Traceback (most recent call last):

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/debuglog.py", line 47, in debug_logging
    yield

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/labextensions.py", line 82, in start
    ans = self.run_task()

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/labextensions.py", line 103, in run_task
    for arg in self.extra_args

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/labextensions.py", line 103, in <listcomp>
    for arg in self.extra_args

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/commands.py", line 316, in install_extension
    return handler.install_extension(extension)

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/commands.py", line 517, in install_extension
    info = self._install_extension(extension, tempdir)

  File "/home/jkbr/miniconda3/lib/python3.7/site-packages/jupyterlab/commands.py", line 1416, in _install_extension
    raise ValueError(msg)

ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=1.0.3 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/application
>=1.0.3 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/apputils

Exiting application: jupyter
```
JupyterLab Version: 1.0.5 installed via miniconda on Linux

**Comment by karosc** [comment]
This should be fixed with the updated package on npm. Note the new package name @karosc/jupyterlab_dracula in the README.

**Comment by Gerenuk** [comment]
The cursor is invisible.

**Comment by karosc** [comment]
Alright. Seems like jupyterlab now as cursor width variables I needed to add. Published v2.0.1 to npm. Cursor is visible on my screen now.
